frontend/static/js/app.js

```diff
@@ -110,11 +110,15 @@ Vue.component("correction-view", {
       this.loadNextCorrection();
     },
     accept: function() {
+      console.log(this.reviewComment)
       this.execAction('approve', {
-      corrected_value: this.correction.val
+      corrected_value: this.correction.val,
+      comment: this.reviewComment ? this.reviewComment : null,
     })},      
     reject: function() { 
-      this.execAction('reject');
+      this.execAction('reject', {

```

Comment:
we should update the 'reset' method to make the comment null so the same comment doesn't appear on next correction